### DISCUSSION TITLE
Open in new tab unPreviewable item (like pdf, docx)

### DISCUSSION THREAD

**Discussion by kamatil-dev** [tusen-ai/naive-ui#4239] [Ideas]
some times user upload documents instead of images and the preview button wont show-up
instead i suggest to open in new tab the items that cannot be previewed

**Comment by kamatil-dev** [comment]
any updates? for pdf or documents preview?

**Comment by jahnli** [comment]
Is it possible to provide a reproduction

**Comment by kamatil-dev** [comment]
@jahnli Ne need for reproduction, it's just the **n-upload** component, when the upload file is a document or pdf, it does not show the *eye icon* (to preview) , so what i want it to show this icon even for unsupported file types and when click on it, it open new tab of this file
![image](https://github.com/tusen-ai/naive-ui/assets/41169051/9bf6d42d-0482-4d53-8a7f-c6bb64307b4e)

**Comment by jahnli** [comment]
Perhaps the design is such that, for file types, opening in a new TAB triggers the browser's download behavior
